dev-docs/FAQ.adoc

```diff
@@ -0,0 +1,11 @@
+= Frequently Asked Questions
+:toc: left
+
+
+== FAQ
+
+This document provides a quick reference for frequently questioned concepts and features in Credentials.
```

Comment:
I was talking to Hoss about this NOTICE.txt situation, and he pointed out that there's a README in `solr/licenses`, and since it already exists that's prob the ideal place to document this.

I was going to push up a PR this morning with two changes:

- add information to the README to cover some of the gaps in adding a dependency
- mention the README specifically in `help/dependencies.txt` so that devs know that it exists and where to look it up for more info.

That's not to say that this couldn't also be covered as a FAQ item.  Maybe it still makes sense to mention here.  Just wanted to let you know that I intend to cover it elsewhere shortly in case that changes your mind on having it here.

Reply:
Check out how I reworked this page.    I think having more pointers/cross referencing is a good thing!

Reply:
LGTM!

Reply:
(Here's the PR I mentioned above, in case you're curious: https://github.com/apache/solr/pull/72)